File: schemas/doituong.py
from pydantic import UUID4, BaseModel, Datetime, Boolean
from typing import Optional
from datetime import datetime


class doituong(BaseModel):
    id : int
    name : Optional[str]
    CMND : Optional[str]
    CCCD : Optional[str]
    Image : Optional[str]
    Ngaysinh : Optional[datetime]
    # True is Nam, False is Nu
    Gioitinh : bool
    Quequan : Optional[str]
    Thongtinbosung : Optional[str]
    SDT : Optional[str]
    KOL: bool
    created_at : datetime
    updated_at : datetime

File: front-end/src/app/pages/Doituong/doituong.ts
export interface doituong{
    client_name: string,
    CMND: string,
    CCCD: string,
    Image:string,
    Ngaysinh: string,
    Gioitinh: boolean,
    Quequan: string,
    Thongtinbosung: string,
    SDT: string,
    KOL: boolean,
}
export interface doituongResponse extends doituong{
    id: string,
    ctnv_id?: number,
    donvi_id?: string,
    donvi_name?: string,
    ctnv_name?: string,
    created_at: string,
    updated_at: string,
}
export interface trangthai {
    created_at: string,
    id: number,
    name: string,
    color: string,
    updated_at: string
}
export interface donvi {
    created_at:string,
    id: string,
    name:string,
    updated_at:string
}
export interface type extends donvi{

}
export interface ctnv extends donvi{

}